.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: php
+
+sudo: true
+
+php:
+- 7.1.3
```

Comment:
Why specifically `7.1.3`? Even the library itself require [7.3](https://github.com/writingink/wink/blob/master/composer.json#L13)?

Reply:
This package requires `^7.1`, not `7.3`.
See https://github.com/writingink/wink/blob/master/composer.json#L13

This package requires `laravel/framework:~5.7`, which requires `7.1.3`
See https://github.com/laravel/framework/blob/5.7/composer.json#L18